packages.yml

```diff
@@ -1,3 +1,6 @@
 packages:
-  - package: fivetran/quickbooks_source
-    version: [">=0.5.0", "<0.6.0"] 
\ No newline at end of file
+# - package: fivetran/quickbooks_source
```

Comment:
Just double-checking, I'll have to comment/uncomment this before releasing?

Reply:
Yes!! You should never release a package that has the hub version commented out and instead points to the git branch.

In order for the BuildKite tests to pass we comment out temporarily. Once the source package is released, we can change the packages.yml and then merge once the tests pass.